.github/json_matrices/build-matrix.json

```diff
@@ -44,23 +44,25 @@
         "NAMED_OS": "linux",
         "ARCH": "arm64",
         "TARGET": "aarch64-unknown-linux-musl",
-        "RUNNER": ["self-hosted", "Linux", "ARM64", "ephemeral"],
+        "CD_TARGET": "aarch64-unknown-linux-gnu",
+        "RUNNER": "ubuntu-latest-arm",
         "CD_RUNNER": "ubuntu-24.04-arm",
         "IMAGE": "node:lts-alpine",
         "CONTAINER_OPTIONS": "--user root --privileged --rm",
-        "PACKAGE_MANAGERS": ["npm"],
+        "PACKAGE_MANAGERS": ["npm", "maven"],
         "languages": []
     },
     {
         "OS": "ubuntu",
         "NAMED_OS": "linux",
         "ARCH": "x64",
         "TARGET": "x86_64-unknown-linux-musl",
+        "CD_TARGET": "aarch64-unknown-linux-gnu",
```

Comment:
cross-compilation? why we can't build x64 client in x64 container on x64 runner and arm client in arm container on arm runner?